data/src/main/java/com/keunsori/data/api/OauthApiService.kt

```diff
@@ -0,0 +1,16 @@
+package com.keunsori.data.api
+
+import com.keunsori.data.data.request.OauthRequest
+import com.keunsori.data.data.request.RefreshRequest
+import com.keunsori.data.data.response.OauthResponse
+import retrofit2.Call
+import retrofit2.http.Body
+import retrofit2.http.POST
+
+interface OauthApiService {
```

Comment:
```suggestion
interface AuthApiService {
```

Oauth는 사용자 인증과 관련된 프로토콜 이름이라, 요기에는 안맞는 이름인 것 같습니다.